qemu-linux-user does not support ptrace, disable tests that depend on it
/proc/self/auxv is not properly abstracted away

Index: elfutils-0.168/tests/Makefile.am
===================================================================
--- a/tests/Makefile.am.orig	2017-08-02 14:06:25.000000000 +0200
+++ b/tests/Makefile.am	2018-03-19 13:51:21.117580061 +0100
@@ -99,7 +99,7 @@
 	run-native-test.sh run-bug1-test.sh \
 	run-debuglink.sh run-debugaltlink.sh run-buildid.sh \
 	dwfl-bug-addr-overflow run-addrname-test.sh \
-	dwfl-bug-fd-leak dwfl-bug-report \
+	dwfl-bug-report \
 	run-dwfl-bug-offline-rel.sh run-dwfl-addr-sect.sh \
 	run-disasm-x86.sh run-disasm-x86-64.sh \
 	run-early-offscn.sh run-dwarf-getmacros.sh run-dwarf-ranges.sh \
@@ -128,8 +128,8 @@
 	run-stack-demangled-test.sh run-readelf-zx.sh run-readelf-zp.sh \
 	run-readelf-dwz-multi.sh run-allfcts-multi.sh run-deleted.sh \
 	run-linkmap-cut.sh run-aggregate-size.sh run-peel-type.sh \
-	vdsosyms run-readelf-A.sh \
-	run-getsrc-die.sh run-strptr.sh newdata elfstrtab dwfl-proc-attach \
+	run-readelf-A.sh \
+	run-getsrc-die.sh run-strptr.sh newdata elfstrtab \
 	elfshphehdr run-lfs-symbols.sh run-dwelfgnucompressed.sh \
 	run-elfgetchdr.sh \
 	run-elfgetzdata.sh run-elfputzdata.sh run-zstrptr.sh \
--- a/tests/backtrace-subr.sh.orig	2017-06-19 16:45:16.000000000 +0200
+++ b/tests/backtrace-subr.sh	2018-03-19 13:49:14.185582823 +0100
@@ -106,6 +106,11 @@
       fi
     ;;
   esac
+
+  if egrep 'qemu: Unsupported syscall' $err; then
+    echo >&2 $testname: qemu-linux-user does not support ptrace
+    exit 77
+  fi
 }
 
 check_core()
